tox.ini

```diff
@@ -30,11 +30,12 @@ match-dir = (?!migrations)
 
 [pytest]
 DJANGO_SETTINGS_MODULE = test_settings
-PYTHONPATH = mock_apps/
 addopts = --cov bulk_grades --cov-report term-missing --cov-report xml
 norecursedirs = .* docs requirements
 
 [testenv]
+setenv =
```

Comment:
looks like this also needs to go in the pii_check tox env.

Also, you can get rid of the django18 targets (might be in the travis config?)